internal/service/advancedclustertpf/data_source.go

```diff
@@ -32,3 +31,9 @@ func (d *ds) Schema(ctx context.Context, req datasource.SchemaRequest, resp *dat
 
 func (d *ds) Read(ctx context.Context, req datasource.ReadRequest, resp *datasource.ReadResponse) {
 }
+
+// TODO: see if resource model can be used instead, probably different only in timeouts
+type ModelDS struct {
```

Comment:
move model to data_source as schema is now auto-generated for singular and plural ds